### DISCUSSION TITLE
Slow editting of large JS file without LSP enabled

### DISCUSSION THREAD

**Discussion by tkroll** [LazyVim/LazyVim#522] [Q&A]
Hello. Loving Lazy. I've learned a lot about Neovim and Lua exploring how it works.

I notice a strange typing delay on a 160KB (6300 line) Javascript file.
The machine is running Ubuntu 22 with 8GB of RAM. I ssh into it.

This is a vanilla LV install. Mason hasn't installed tsserver and htop confirms it's not loaded.
I tried my own config with tsserver running and the slow down was not present.

Any ideas what this could be?

Thanks

**Comment by amaanq** [comment]
nvim-ts was likely the culprit w/ injections for gql comments & template strings, I updated that a week or two ago so just update & it should be fine now.
